templates/main/index.html.ep

```diff
@@ -19,7 +19,6 @@
     </div>
 </div>
 
-<a name="build-results"/>
```

Comment:
that's what I had in before but @coolo wanted no increase of indendation (see discussion above)